README.md

```diff
@@ -318,7 +318,7 @@ This is done by certbot.
 In order for this state to run successfully, it is important that the proxy is already accessible from the Internet and the DNS entries are set.  
```

Comment:
In line 316, it should say '**proxy**-enable-https', right?

Reply:
Done!